README.md

```diff
@@ -1,27 +1,22 @@
-# Appstart Manual
-
-## by: gouzenko
-
+# Appstart
 
 ## Introduction
 
-Appstart is a tool that allows Managed VM applications to be deployed locally
-for testing purposes. It provides access to Google Cloud Platform stubs
-via an API server that runs in its own container. Appstart relies on docker-py
-to manage the API server container and the user's application container.
-
-This is not an official Google product (experimental or otherwise), it is just
-code that happens to be owned by Google.
+appstart is a local tool that allows Managed VM applications to be deployed into
+a local Docker cluster in which App Engine services such as Task Queues and
+Datastatore are emulated. It can be useful for advanced users who wish to perform
+additional local testing before deploying applications to App Engine.
 
+*Important: This project is experimental and is not officially supported by the 
+Google Cloud Platform.*
 
-## Installation
+## Installation (Linux)
```

Comment:
Doesn't work on Mac. Needs to be tested on Windows.

On Wed, Aug 26, 2015, 08:41 mindhog <notifications@github.com> wrote:

> In README.md
> <https://github.com/GoogleCloudPlatform/appstart/pull/1#discussion_r37997622>
> :
>
> >
> > -## Installation
> > +## Installation (Linux)
>
> Should be Linux and Mac, right? (I don't know how this works with Windows,
> or whether all of the Docker support is even there yet)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/appstart/pull/1/files#r37997622>.
>